### ISSUE TITLE
Detailed listing of SMIRNOFF99Frosst parameterization

### ISSUE THREAD

**Issue by slochower** [slochower/smirnoff-host-guest-manuscript#8]
I decided to skip a detailed listing of what I did, and instead link to the Jupyter notebook I used, because these instructions may change with future toolkit versions, and in fact, it is already out of date in some ways. If this is a good way to cite this, I can upload that GitHub repository to Zenodo, get a DOI, and cite the DOI. It may also be feasible to include this as SI, but that can get messy for anything other than trivial code listings.

**Comment by davidlmobley** [comment]
I agree with this approach. Depositing the code is often superior to listing a bunch of details. Though, you may want to list crucial details/main points/anything odd or especially important.
